raiwidgets/raiwidgets/fairness_dashboard.py

```diff
@@ -82,14 +82,12 @@ def __init__(
                 metrics_module.probability_methods,
         }
 
-        if locale is not None:
```

Comment:
What if locale is None?

Reply:
locale is passed into super (dashboard) class and get transmitted to ui through "config" object, this part is not used in ui, that's why remove.
If locale is none, we use en-us by default